rollup.config.js

```diff
@@ -0,0 +1,86 @@
+import babel from 'rollup-plugin-babel';
+import replace from 'rollup-plugin-replace';
+import uglify from 'rollup-plugin-uglify';
+import pkg from './package.json';
+
+const mergeAll = objs => Object.assign({}, ...objs);
+
+const commonPlugins = [
+  babel({
+    exclude: 'node_modules/**',
+    plugins: ['external-helpers'],
+  }),
+];
+
+const configBase = {
+  input: 'src/index.js',
+  external: [
+    ...Object.keys(pkg.dependencies || {}),
+    ...Object.keys(pkg.peerDependencies || {}),
+  ],
+  plugins: commonPlugins,
+};
+
+const umdConfig = mergeAll([
+  configBase,
+  {
+    output: {
+      file: `dist/${pkg.name}.js`,
+      format: 'umd',
+      name: 'FlipMove',
+      globals: {
+        react: 'React',
+        'react-dom': 'ReactDOM',
+      },
+    },
+    external: Object.keys(pkg.peerDependencies || {}),
+  },
+]);
+
+const devUmdConfig = mergeAll([
+  umdConfig,
+  {
+    plugins: umdConfig.plugins.concat(
+      replace({
+        'process.env.NODE_ENV': JSON.stringify('development'),
+      }),
+    ),
+  },
+]);
+
+const prodUmdConfig = mergeAll([
+  umdConfig,
+  {
+    output: mergeAll([
+      umdConfig.output,
+      { file: umdConfig.output.file.replace(/\.js$/, '.min.js') },
+    ]),
+  },
+  {
+    plugins: umdConfig.plugins.concat(
+      replace({
+        'process.env.NODE_ENV': JSON.stringify('production'),
+      }),
+      uglify({
+        compress: {
+          pure_getters: true,
+          unsafe: true,
+          unsafe_comps: true,
+          warnings: false,
```

Comment:
what are `unsafe_comps` and `warnings`? Couldn't find it in the Uglify docs

Reply:
from the UglifyJS Readme.md

> unsafe_comps (default: false) -- Reverse < and <= to > and >= to allow improved compression. This might be unsafe when an at least one of two operands is an object with computed values due the use of methods like get, or valueOf. This could cause change in execution order after operands in the comparison are switching. Compression only works if both comparisons and unsafe_comps are both set to true.

warnings can be removed though as `false` is actually the default

> warnings (default: false) -- display warnings when dropping unreachable code or unused declarations etc.

Reply:
:+1: thanks. Not sure how I missed it, must've been looking at different docs. Both settings makes sense!